### PR TITLE
Servicenow support

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Support for Service now

# Delete branch after merge
YES

# NEWS file entries
## New Features
* ServiceNow support

### CODE REVIEW

**Comment by julsemaan** [comment]
@fdurand 
Still 8.1 material ? If so, then @satkunas should get onto the review

**Comment by extrafu** [comment]
@fdurand Update for the new GUI.

**Comment by fdurand** [comment]
New admin ok, missing documentation.

**Comment by fdurand** [comment]
Rebased

**Comment by extrafu** [comment]
@julsemaan We're waiting from documentation for this (from customer using it). I think we should go ahead and merge that anyway - it's a no-risk/no-brainer thing and we could always find someone else willing to share the doc with us. We just need to file an issue right after the merge that we must have the doc for this.

File: html/captive-portal/templates/provisioner/servicenow.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="layout--center">
+  <h5 class="u-mb--">[% i18n("To complete your network activation we need to verify this is a corporate computer.") %]</h5>
```
Comment (extrafu) [inline html/captive-portal/templates/provisioner/servicenow.html]: .... we need to verify that your device is corporate-owned.

File: html/captive-portal/templates/provisioner/servicenow.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <p>[% i18n("Once the verification is done, click 'Continue' to activate your network connection") %]</p>
```
Comment (extrafu) [inline html/captive-portal/templates/provisioner/servicenow.html]: Once the verification has completed, ...

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/ProvisioningsList.vue [hunk 3]
```diff
@@ -27,6 +27,7 @@
           <b-dropdown-item :to="{ name: 'newProvisioning', params: { provisioningType: 'symantec' } }">Symantec App Center</b-dropdown-item>
           <b-dropdown-item :to="{ name: 'newProvisioning', params: { provisioningType: 'windows' } }">Windows</b-dropdown-item>
           <b-dropdown-item :to="{ name: 'newProvisioning', params: { provisioningType: 'intune' } }">Microsoft Intune</b-dropdown-item>
+          <b-dropdown-item :to="{ name: 'newProvisioning', params: { provisioningType: 'servicenow' } }">Service Now</b-dropdown-item>
```
Comment (extrafu) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/ProvisioningsList.vue]: Service Now -> ServiceNow

**Comment by extrafu** [comment]
@fdurand Minor typos there and there - fix them and I'll merge it right away. Good work on this.

**Comment by fdurand** [comment]
@extrafu done

**Pull request merged**